packages/client/test/client.spec.ts

```diff
@@ -95,20 +95,35 @@ describe('test store localStorage', () => {
     localStorage.clear();
   });
   it('test storing signature in localStorage not using password', () => {
-    expect(getSignature(StorageType.LocalStorage)).to.be.null;
-    storeSignature(properties, sign, StorageType.LocalStorage);
-    const storageData = getSignature(StorageType.LocalStorage);
+    expect(getSignature('test', StorageType.LocalStorage)).to.be.null;
+    storeSignature('test', properties, sign, StorageType.LocalStorage);
+    const storageData = getSignature('test', StorageType.LocalStorage);
     expect(storageData).to.be.not.null;
-    expect(storageData.properties).to.be.deep.equal(properties);
-    expect(storageData.sign).to.be.deep.equal(sign);
+    expect(storageData?.properties).to.be.deep.equal(properties);
+    expect(storageData?.sign).to.be.deep.equal(sign);
   });
   it('test storing signature in localStorage using password', () => {
-    expect(getSignature(StorageType.LocalStorage, 'abc')).to.be.null;
-    storeSignature(properties, sign, StorageType.LocalStorage, 'abc');
-    const storageData = getSignature(StorageType.LocalStorage, 'abc');
+    expect(getSignature('test', StorageType.LocalStorage, 'abc')).to.be.null;
+    storeSignature('test', properties, sign, StorageType.LocalStorage, 'abc');
+    const storageData = getSignature('test', StorageType.LocalStorage, 'abc');
     expect(storageData).to.be.not.null;
-    expect(storageData.properties).to.be.deep.equal(properties);
-    expect(storageData.sign).to.be.deep.equal(sign);
+    expect(storageData?.properties).to.be.deep.equal(properties);
+    expect(storageData?.sign).to.be.deep.equal(sign);
+  });
+  it('test storing signature in localStorage using different key', () => {
+    expect(getSignature('test1', StorageType.LocalStorage)).to.be.null;
+    storeSignature('test1', properties, sign, StorageType.LocalStorage, 'abc');
+    expect(getSignature('test2', StorageType.LocalStorage)).to.be.null;
+    storeSignature('test2', properties, sign, StorageType.LocalStorage);
+    const storageData1 = getSignature('test1', StorageType.LocalStorage, 'abc');
+    expect(storageData1).to.be.not.null;
+    expect(storageData1?.properties).to.be.deep.equal(properties);
+    expect(storageData1?.sign).to.be.deep.equal(sign);
+    const storageData2 = getSignature('test2', StorageType.LocalStorage);
+    expect(storageData2).to.be.not.null;
+    expect(storageData2?.properties).to.be.deep.equal(properties);
+    expect(storageData2?.sign).to.be.deep.equal(sign);
+    expect(() => getSignature('test2', StorageType.LocalStorage, 'abc')).to.throw(Error);
```

Comment:
Test case to check whether the value is stored separately if the key value is different.